File: src/main/java/com/dennisturco/dto/QuotationResponseDTO.java
package com.dennisturco.dto;

import java.time.LocalDate;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class QuotationResponseDTO {
    private Long id;
    private Long customerId;
    private String customerSurname;
    private Long quotationStatusId;
    private String quotationStatusName;
    private String number;
    private Float amount;
    private Integer vatPercentage;
    private Integer discountPercentage;
    private String description;
    private String notes;
    private LocalDate creationDate;
    private LocalDate lastUpdateDate;
    private LocalDate issueDate;
    private LocalDate validityDate;
}

File: GestioPro.Common/DTOs/QuotationResponseDTO.cs
using GestioPro.Common.Enums;

namespace GestioPro.Common.DTOs;

public record QuotationResponseDTO(
    long Id,
    long CustomerId,
    string CustomerName,
    QuotationStatus QuotationStatus,
    string Number,
    string Title,
    float Amount,
    int VatPercentage,
    int DiscountPercentage,
    string? Description,
    string? Notes,
    DateTimeOffset CreationDate,
    DateTimeOffset LastUpdateDate,
    DateOnly? IssueDate,
    DateOnly? ValidityDate,
    bool IsDisabled
);
